docs/writing_review_screen.md

```diff
@@ -0,0 +1,240 @@
+    id: revisit users
+    continue button field: users.revisit
+    question: |
+      Edit your answers about Users
+    subquestion: |
+      ${ users.table }
+
+      ${ users.add_action() }
+    ---
+    ...
+    ```
+
+    Each block has the table and lets people edit items in the lists.
+
+4. The generated tables for the lists.
+
+    ```yml
+    table: children.table
+    rows: children
+    columns:
+      - Birthdate: |
+          row_item.birthdate if hasattr(row_item, 'birthdate') else ''
+    edit:
+      - birthdate
+    ```
+
+### Step 3: Edits to the Review Screen Generator
+
+You'll want to change some aspects of the generated review screen. Note that if any errors happen in a specific edit element on the review screen, that element won't appear. It also won't appear if the attribute or variable to be edited doesn't exist. You should make sure that it does.
+
+Some common issues to the generated review screen that I personally have run into before:
+
+1. Add tables for people lists, like `users` or `other_parties`. The below block is a good start.
+
+```yml
+generic object: ALPeopleList
+table: x.table
+rows: x
+columns:
+  - Name: |
+      row_item.name if defined('row_item.name.first') else ''
+  - Address: |
+      row_item.address if defined('row_item.address.address') else ''
+  - Phone number: |
+      row_item.phone_numbers()
+edit:
+  - name.first
+  - address.address
+  - phone_number
+delete buttons: False
+```
+
+2. Remove tables for attachments and other non-people objects, if any were generated. Tables are only needed for information that the user enters themselves, which the review screen generator can't determine yet.
+3. Remove edit questions about kickout screens. Sometimes, you will have questions that do set a variable in the interview, but it's only use is to kickout a user if their problem isn't covered by the form. Here's an example of one:
+
+```yml
+---
+id: Is this a probate case
+question: |
+  Are you Filing in a Probate Case?
+subquestion: |
+  Is the case a probate case? Examples of probate cases include cases to distribute property after someone dies, guardianships, and conservatorships. 
+fields:
+  - Is your case a probate case?: probate_case
+    datatype: yesnoradio
+---
+event: probate_case_exit
+question: |
+  You cannot continue to use the tool.
+subquestion: |
+  Based on your answer, you can't use this tool to make a Notice of Hearing and Motion. Probate cases have different procedural rules than other cases. You may want to speak to a lawyer.
+  
+  If you made a mistake, click the **Undo** button. Otherwise, click the **Exit** button to return to Michigan Legal Help.
+buttons:
+  - Exit: exit
+    url: https://michiganlegalhelp.org
+```
+
+If editing the value of a variable like this would only lead the person to a kickout screen and not change any other logic of the interview, then the generated review section should be removed.
+
+
+### Step 4: Things to add to the review screen
+
```

Comment:
This isn't such a helpful heading. What about directly describing the things to add or customize?

E.g., "Add logic for follow-up questions when an answer is changed by the user"